build.gradle

```diff
@@ -33,6 +33,7 @@ repositories {
     jcenter()
     maven { url "http://dl.bintray.com/kotlin/ktor" }
     maven { url "http://dl.bintray.com/kotlin/exposed" }
+    maven { url "https://dl.bintray.com/kotlin/squash" }
```

Comment:
Do we need the squash repository? I tried to compare squash and exposed, seems they are almost the same.